test/e2e/data/infrastructure-aws/machine-pool/machine-pool.yaml

```
@@ -1,5 +1,5 @@
 ---
-apiVersion: cluster.x-k8s.io/v1alpha4
+apiVersion: exp.cluster.x-k8s.io/v1alpha4
```

MachinePool API group is no longer exp.
```suggestion
apiVersion: cluster.x-k8s.io/v1alpha4
```